bigtable-client-core-parent/bigtable-hbase/src/main/java/com/google/cloud/bigtable/hbase/BatchExecutor.java

```diff
@@ -135,19 +135,19 @@ public final void onFailure(Throwable throwable) {
   }
 
   protected static class BulkOperation {
-    private BulkMutation bulkMutation;
+    private BulkMutationWrapper bulkMutationWrapper;
```

Comment:
Shall I consider naming this variable shorter? say `clientWrapper` or `bulkWrapper`